ci/requirements/environment-dev.yml

```diff
@@ -2,7 +2,7 @@ name: climpred-dev
 channels:
   - conda-forge
 dependencies:
-  - python=3.6
+  - python=3.*
```

Comment:
I can't seem to find this notation anywhere. What does it actually do? If I run `conda env update -f environment-dev.yml` which python does it choose?

Reply:
you can check in travis. for travis -python: 3.6 it takes py36, for 3.7 it takes 3.7

Reply:
I still think its better to have just one env file instead of managing many. xr needs to that but they also track their latest supper versions in pandas numpy and so on